File: Programers/Level1/이상한문자만들기.java
package Programers.Level1;

public class 이상한문자만들기 {

    public String solution(String s) {
        String answer = "";
        int cnt = 2;
        for(int i=0; i < s.length(); i++){
            String ss = s.charAt(i)+"";
            if(s.charAt(i) == ' '){
                cnt = 2;
                answer += " ";
                continue;              
            }
            
            else if(cnt % 2 == 0)
                answer += ss.toUpperCase();
            else
                answer += ss.toLowerCase();
            
            cnt++;            
        }        
        return answer;
    }

}

File: Programers/Level1/이상한문자만들기.py
def solution(s):
    answer = ''
    
    str_list = list(s)
    cnt = 2
    for i in range(len(str_list)) :
        if(str_list[i] == ' ') :
            cnt = 2
            answer += str_list[i]
            continue
        elif(cnt % 2 == 0) :
            answer += str_list[i].upper()
        else :
            answer += str_list[i].lower()
        
        cnt +=1
        
    return answer